packages/scenes-app/src/demos/adhocFiltersDemo.tsx

```diff
@@ -27,15 +26,16 @@ export function getAdhocFiltersDemo(defaults: SceneAppPageState) {
         getScene: () => {
           return new EmbeddedScene({
             ...getEmbeddedSceneDefaults(),
-            controls: [
-              new AdHocFilterSet({
-                name: 'Filters',
-                // Only want keys for this series
-                baseFilters: [{ key: '__name__', operator: '=', value: 'ALERTS', condition: '' }],
-                datasource: { uid: 'gdev-prometheus' },
-              }),
-              ...getEmbeddedSceneDefaults().controls,
-            ],
+            $variables: new SceneVariableSet({
+              variables: [
+                new AdHocFiltersVariable({
+                  name: 'Filters',
+                  // Only want keys for this series
+                  baseFilters: [{ key: '__name__', operator: '=', value: 'ALERTS', condition: '' }],
```

Comment:
```suggestion
                  baseFilters: [{ key: '__name__', operator: '=', value: 'ALERTS' }],
```
`condition` is optional and it [is deprecated](https://github.com/grafana/grafana/commit/6a37a56d68f4c47dd6880c457c31eb07c9a96b04).

We need to update the version of `@grafana/data` to the latest.